omero/developers/Web/CreateApp.rst

```diff
@@ -4,296 +4,145 @@ Creating an app
-            {% endfor %}
-       </body>
-       </html>
+    Here we use single quotes around double quotes.
 
+::
 
-Viewing the page at http://localhost:4080/<appname>/stack_preview/<image-id>/
-should give you the image name and 5 planes from the Z stack. You will notice
-that we are using the ``webgateway`` to handle the image rendering using a URL
-auto-generated by Django - see :doc:`/developers/Web/WebGateway`.
+    $ bin/omero config append omero.web.apps '"minimal_webapp"'
 
-Resources for writing your own code
------------------------------------
 
-The `OMERO.webtest <https://github.com/openmicroscopy/omero-webtest/>`_ app
-has a number of examples. Once installed, you can go to the webtest
-homepage e.g. `<http://localhost:4080/webtest>`_ and you will
-see an introduction to some of them. This page tries to find random
-images and datasets from your OMERO server to use in the webtest examples.
+Exploring the app
+-----------------
 
+The ``minimal_webapp`` code is well documented to explain
+how the app is working.
+Briefly, the app supports a single URL defined in
+``minimal_webapp/urls.py`` which maps to the ``index`` function
+within ``minimal_webapp/views.py``. This uses the connection to
+OMERO, ``conn`` to load the current user's name and passes this
+to the ``index.html`` template to render the page.
 
-.. _jquery_and_jquery_ui:
+This page also includes the static ``app.js`` and ``app.css`` files.
+JavaScript is used to load Projects from the :doc:`/developers/json-api` and
+display them on the page.
 
-Using jQuery and jQuery UI from OMERO.web
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Create an app from the template example
+---------------------------------------
 
-OMERO.web uses the `jQuery <https://jquery.com/>`_ and `jQuery UI <https://jqueryui.com/>`_ 
-javascript libraries extensively.
-If you need these libraries, you can include the OMERO.web versions of
-these libraries in your own pages. The alternative is to provide a specific
-version of jQuery or jQuery UI in your own app if, for example, you think
-that a version change may cause problems in your code.
-If you need to make use of these resources in your own pages, you can
-add the following statements to the ``<head>`` of your page templates::
+If you want to create your own app, you can use the example
+as a template.
 
-    <!-- jQuery -->
-    {% include "webgateway/base/includes/script_src_jquery.html" %}
+Go to the template repository
+`omero-web-apps-examples <https://github.com/will-moore/omero-web-apps-examples>`_.
+Click 'Use this template' as `described here
+<https://help.github.com/en/articles/creating-a-repository-from-a-template>`_
+and choose a name for your new repo, for example ``my_app``.
 
-    <!-- jQuery UI - includes js and css -->
-    {% include "webgateway/base/includes/jquery-ui.html" %}
+Go to the directory where you want your app to live and clone it.
+Then run as above using a different ``appdir`` variable:
```

Comment:
This is the only place where I got a bit lost. 

``as above`` -> ``as above in section {link to the section "Run your app with OMERO.web in a Docker container" (e.g  https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/developers/Web/CreateApp.html#run-your-app-with-omero-web-in-a-docker-container)